msteams-platform/overview.md

```diff
@@ -1,100 +1,55 @@
 ---
-title: Microsoft Teams developer platform
+title: Developer platform
 description: Overview page describing the Microsoft Teams developer platform
 keywords: teams overview
 ---
 
-# Overview of the Microsoft Teams developer platform
+# The Microsoft Teams developer platform
 
-Microsoft Teams is a chat-based workspace in Office 365 that integrates with the apps and services teams use to get work done together.  The Microsoft Teams developer platform makes it easy for you to integrate your own service, whether you develop custom apps for your enterprise or SaaS applications for teams around the world.
+Microsoft Teams is a chat-based workspace in Office 365 that integrates with the apps and services that teams use to get work done together. The Microsoft Teams developer platform makes it easy for you to integrate your own service, whether you develop custom apps for your enterprise or SaaS applications for teams around the world.
```

Comment:
Again instead of teams, do you want to consider 'groups of people' in the first line?

Reply:
Good idea. I'll even go further and say simply "people" because of the word "together" at the end of the sentence. (After all, "people getting work done together" is pretty much the definition of "team.")